app/utils/valid_account_id.js

```diff
@@ -10,15 +10,38 @@ const userServicesContainsGatewayAccount = function userServicesContainsGatewayA
 }
 
 const liveUserServicesGatewayAccounts = async function liveUserServicesGatewayAccounts (user) {
```

Comment:
Could this file be renamed from `valid_account_id` while you're here, as it does a bit more than validate an account id. `service_gateway_accounts.js` or something, I dunno?

Reply:
Also there's no unit test for this function, reckon we should add one now that it does more?